src/keycloak/chart/values.yaml

```diff
@@ -149,17 +153,18 @@ service:
   # Session affinity config
   sessionAffinityConfig: {}
 
+# Connection information for external postgres database
 postgresql:
-  # PostgreSQL User to create
-  username: keycloak
-  # PostgreSQL Password for the new user
-  password: keycloak
-  # PostgreSQL Database to create
-  database: keycloak
-  # PostgreSQL host
-  host: postgresql
-  # PostgreSQL port
-  port: 5432
+  # The username of the database user
+  username: ""
+  # The password of the database user
+  password: ""
+  # Database name
+  database: ""
+  # URL for the database
+  host: ""
+  # Port the database is listening on (default 5432)
+  port: ""
```

Comment:
I think I missed this one before, how do you feel about leaving 5432 here as the default and removing port from the helper checks?

Reply:
I'm good with that.